101-machine-learning-compute-create-computeinstance/azuredeploy.parameters.json

```diff
@@ -1,18 +1,18 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
```

Comment:
I think we need to roll back changes to this file, some of these are still required... also not clear on the scenario you're trying to enable.

Reply:
Made changes to the parameter file. User needs to fill in workspace name and Azure region to create the compute instance in. Please review